cmd/kanikoExecute.go

```diff
@@ -404,10 +416,11 @@ func runKaniko(dockerFilepath string, buildOptions []string, readDigest bool, ex
 }
 
 type multipleImageConf struct {
-	ContextSubPath     string `json:"contextSubPath,omitempty"`
-	ContainerImageName string `json:"containerImageName,omitempty"`
-	ContainerImageTag  string `json:"containerImageTag,omitempty"`
-	ContainerImage     string `json:"containerImage,omitempty"`
+	ContextSubPath          string `json:"contextSubPath,omitempty"`
+	ContainerDockerfilePath string `json:"containerDockerfilePath,omitempty"`
```

Comment:
The convention we tried to establish with [`cnbBuild.multipleImages`](https://github.com/SAP/jenkins-library/blob/c1bb7f86bcaa14e4dcc33dbac6c9034a7c49e207/resources/metadata/cnbBuild.yaml#L261-L292)was that you can define meaningful parameters either on the root parameters for the step or inside `multipleImages`.

I would suggest that `kanikoExecute` follows this precedent and just allows `kanikoExecute.multipleImages.dockerfilePath` to override [`kanikoExecute.dockerfilePath`](https://github.com/SAP/jenkins-library/blob/48e5bf35c911b35d82d879254a16323497ee3840/resources/metadata/kanikoExecute.yaml#L268C1-L277) instead of introducing a different name in that scope.

cc @c0d1ngm0nk3y @pbusko @anilkeshav27